src/base/preferences.cpp

```diff
@@ -1364,14 +1374,14 @@ void Preferences::setListenIPv6(bool enable)
     setValue("Preferences/Connection/InterfaceListenIPv6", enable);
 }
 
-QString Preferences::getNetworkAddress() const
+QString Preferences::getAnnounceAddress() const
 {
-    return value("Preferences/Connection/InetAddress").toString();
+    return value("Preferences/Connection/AnnounceAddress").toString();
```

Comment:
Don't change the path. Users of older versions will lose their settings. Leave comment here for the coder to understand what this represents instead.